NEXTCLOUD.cmake

```diff
@@ -34,8 +34,9 @@ option( WITH_PROVIDERS "Build with providers list" ON )
 
 
 ## Theming options
-set( APPLICATION_WIZARD_HEADER_BACKGROUND_COLOR "#0082c9" CACHE STRING "Hex color of the wizard header background")
-set( APPLICATION_WIZARD_HEADER_TITLE_COLOR "#ffffff" CACHE STRING "Hex color of the text in the wizard header")
+set(NEXTCLOUD_BACKGROUND_COLOR "#0082c9" CACHE STRING "Default Nextcloud background color")
```

Comment:
I know we all hate the naming of those variables but I wouldn't do this, or if really I'd keep the cmake variables unchanged but change the defines for the C++ side.

This will be hard to coordinate with the brander side otherwise and this will add yet another one of those "per version" change in the brander job code. It's going to be painful.

Also a word of caution: this is getting more and more remote of the initial PR intent, it's slowly getting bigger and less focused this is how quality issues manage to go under the radar.

Reply:
We were discussing this and Tobias said it's fine to change.

Reply:
Still, separate PR would have been better for my brain. ;-)

Reply:
I now reverted these changes. Will do a separate pr :)